task.sql

```diff
@@ -1,3 +1,16 @@
 USE ShopDB; 
 
 -- Create your stored procedure here
+DELIMITER //
+CREATE PROCEDURE get_warehouse_product_inventory(
+    IN HouseID INT
+    )
+BEGIN
+    SELECT P.Name, WarehouseAmount
```

Comment:
Why you don't use the full name of the field like Products.Name? It makes code more readable